tests/acceptance/features/bootstrap/FeatureContext.php

```diff
@@ -3299,6 +3326,14 @@ public function substituteInLineCodes(
 					"getLastPublicShareToken"
 				],
 				"parameter" => []
+			],
+			[
+				"code" => "%group_id%",
+				"function" => [
+					$this,
+					"getUUIDv4Regex"
+				],
+				"parameter" => []
```

Comment:
fetch the  pattern from static class `GraphHelper` web already have this function there right?